_usage/interoperability.md

```diff
@@ -0,0 +1,109 @@
+
+Since the Babelfish database is ultimately a PostgreSQL database, it is technically possible to access it from the PostgreSQL port and run PostgreSQL SQL statements against it. However, users should tread carefully when accessing the Babelfish database through PostgreSQL, as there are various complexities to be considered. 
+
+From a Babelfish perspective, it is simplest to avoid accessing the Babelfish database from PostgreSQL altogether. However, there may be valid reasons for wanting to use PostgreSQL functionality anyway. For example, when a particular aspect of T-SQL is not supported by Babelfish, it might be possible to implement that aspect natively in PostgreSQL so as to allow migrating the application to Babelfish PostgreSQL. However, this comes with certain implications and risks, as described below.
+
+The interoperability considerations apply whenever the execution context is not pure TDS/T-SQL, i.e. when PostgreSQL SQL statements are executed in a Babelfish context. This is the case when:
+
+- In a PostgreSQL connection, PostgreSQL statements are executed against Babelfish-created objects (e.g. tables, views, procedures). 
+- In a TDS connection, T-SQL statements are executed against PostgreSQL-created objects (e.g. tables, views, procedures) or when PostgreSQL built-in functions are called from T-SQL. Based on customer feedback, this seems to be the more common of these scenarios.
+
+In both cases, this effectively means the SQL dialect is temporarily (and transparently) switched in the session while accessing/executing the objects created in a different SQL dialect (this does not apply to accessing a table).
+
+When one of the above cases occurs in a Babelfish-based application, one or more of the following interoperability aspects may need to be considered. It is the responsibility of the Babelfish user to determine whether an application with such a mixed-SQL dialect scenario works as expected. Also, the user should anticipate potential issues when upgrading to future versions of Babelfish; such issues may need to be resolved by the user themselves, as AWS may not be able to support such mixed-SQL dialect cases.
+
+- **Schema names**
+     Babelfish emulates the SQL Server multi-DB structure by flattening the various SQL Server databases into schemas inside the single `babelfish_db` PostgreSQL database. For example, this means that a T-SQL table created as `mydb.dbo.mytable` and a procedure created as `yourdb.dbo.yourproc` must be referenced from PostgreSQL as table `mydb_dbo.mytable` and procedure `yourdb_dbo.yourproc`, respectively [see Using Babelfish with a single database or multiple databases](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/babelfish.html).
+     In addition, SQL Server uses different rules than PostgreSQL for resolving an object name without a qualifying schema name. 
```

Comment:
the link to the single/multi explanation is not correct. SHould be: https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/babelfish-architecture.html